src/test/java/subway/RestAssuredTest.java

```diff
@@ -16,7 +17,9 @@ public class RestAssuredTest {
     @Test
     void accessGoogle() {
         // TODO: 구글 페이지 요청 구현
-        ExtractableResponse<Response> response = null;
+        ExtractableResponse<Response> response = RestAssured.given()
+                .when().get("https://google.com")
+                .then().log().all().statusCode(200).extract();
```

Comment:
아래에서 별도로 응답 코드에 대한 검증을 하고 있어서 이 부분은 제거해도 좋을 것 같아요!

```suggestion
        ExtractableResponse<Response> response = RestAssured.given()
                .when().get("https://google.com")
                .then().log().all().extract();
```